src/System.Runtime.Serialization.Xml/tests/Configurations.props

```diff
@@ -5,6 +5,7 @@
       netstandard;
       netcoreapp;
       uap;
+      netfx;
```

Comment:
- **Separate project**: I've created new folder Canonicalization under tests, move case files there and created new csproj in it. Do you mean that? Due to file name length limitation, the project was named "System.Runtime.Serialization.Xml.CN.Tests.csproj", with test class's namespace name remain System.Runtime.Serialization.Xml.Canonicalization.Tests, does it look ok?
- **About netfx prop**: if delete the netfx, build with targetgroup netfx would fail with error about Transform & XmlDsigExcC14NTransform could not be found. Add it test can pass on netfx. I've added the config to test the case, it's then deleted in this PR and the case is now marked with active issue.
- **ReflectionOnly project needed**? like System.Runtime.Serialization.Xml.ReflectionOnly.Tests